docs/upgrades.md

```diff
@@ -61,3 +61,68 @@ sequenceDiagram
        end
     end
 ```
+
+### Introducing package manifest
+
+Starting from version 8.13.0 an additional file `manifest.yaml` is present in elastic-agent packages.
```

Comment:
Documentation is great, let's add why we did this. To allow releasing versions of the agent package where only the component versions change, with the agent commit unchanged.

Reply:
Have a look at b87a013af7e3719dde052c5375eba0d92d95f26c
I tried adding a bit more context on that

Reply:
Perfect, thanks!